go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -225,5 +263,5 @@ func (nt *nodeTracker) allNodes() []nodeInfo {
 	// so that other operations that consume this data can just do a DeepEquals of things
 	// (e.g. LB routers + switches) without having to do set arithmetic
 	sort.SliceStable(out, func(i, j int) bool { return out[i].name < out[j].name })
-	return out
+	return out, *nt.nodeIPv4Template, *nt.nodeIPv6Template
```

Comment:
This is not really a copy and can still change. Should we copy?

Reply:
Yes, we should.  Done.